evt.js

```diff
@@ -64,6 +65,16 @@
     }
   }
 
+  // If there is an error listner, delegate to that, otherwise just log the
+  // errors, so that emit notifications to other listeners still work.
+  function emitError(err) {
+    if (evt._events.hasOwnProperty('error')) {
+      evt.emit('error', err);
+    } else {
+      console.error(err + '\n' + err.stack);
```

Comment:
For console.error, it's probably preferable to do something like `console.error(err, '\n' + err.stack);` so that devtools' webconsole gets the raw error object so it can be exposed as an inspectable object or have other magic applied to it.  (We could also go further if we're assuming more non-spidermonkey JS engines.  For SM since I think we still spec stack to be a string, stringifying it doesn't hurt it, but V8 does do that fancy object stack.  Although maybe their devtools would key off the object anyways.)